client/additional-methods-setup/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -39,7 +39,7 @@ import CurrencyInformationForMethods from './currency-information-for-methods';
 import { getMissingCurrenciesTooltipMessage } from 'wcpay/multi-currency/missing-currencies-message';
 import { upeCapabilityStatuses, upeMethods } from '../constants';
 import paymentMethodsMap from '../../payment-methods-map';
-import ConfirmPaymentMethodActivationModal from 'wcpay/payment-methods/activation-modal';
```

Comment:
It's unfortunate that this `add-payment-methods-task` component needs to tap into another component's directory.
It was like this before, and it still is the case.
If I were to refactor it too, this would have been a much larger PR (on top of all the other changes that preceded it in https://github.com/Automattic/woocommerce-payments/pull/9261 https://github.com/Automattic/woocommerce-payments/pull/9265 https://github.com/Automattic/woocommerce-payments/pull/9255 https://github.com/Automattic/woocommerce-payments/pull/9262 ).